org-roam.el

```diff
@@ -140,6 +140,7 @@ Performs a database upgrade when required."
            (init-db (not (file-exists-p db-file))))
       (make-directory (file-name-directory db-file) t)
       (let ((conn (emacsql-sqlite db-file)))
+	(set-process-query-on-exit-flag (emacsql-process conn) nil)
```

Comment:
Indentation seems a bit off.

Reply:
> Indentation seems a bit off.

I used (default) <kbd>Tabs</kbd> and just amended the commit to  `untabify`.

A global `.editorconfig` would be nice to prevent this issues.

Reply:
Sorry it was right, it looked a bit off on my phone, which was why I resolved it after.